scripts/generate_env.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+CONFIG_FILE="env/openpaas.js"
+
+mkdir -p env
+
+# Recreate config file
+rm -rf ./$CONFIG_FILE
+touch ./$CONFIG_FILE
+
+# Add assignment
+echo "window.openpaas = {" >> ./$CONFIG_FILE
+
+for key in $(env | awk -F "=" '{print $1}' | grep "OPENPAAS.*")
+# key is something like OPENPAAS_API_URL
+do
+    # Append configuration property to JS file
+    echo "  $key: \"$(printenv $key)\"," >> ./$CONFIG_FILE
+done
+
+echo "}" >> ./$CONFIG_FILE
```

Comment:
Why not having a sample file and use `sed` to replace values?

Reply:
Then we have to add that sample file in our git repository, which is redundant.
Creating a new file each time a container is started is much cleaner

Reply:
Moreover, if we have a sample file (`env/openpaas.js`), we also need `copy-webpack-plugin` to copy this file (after `sed` with correct values) to the build.
Christophe also said no to this https://chat.linagora.com/openpaas/pl/14h5myek47r7zdu86j7syaws8h :D

Reply:
Ok